modules/frontend/tag_handlers.go

```diff
@@ -137,10 +153,21 @@ func newTagsV2StreamingGRPCHandler(cfg Config, next pipeline.AsyncRoundTripper[c
 	}
 }
 
-func newTagValuesStreamingGRPCHandler(cfg Config, next pipeline.AsyncRoundTripper[combiner.PipelineResponse], apiPrefix string, o overrides.Interface, logger log.Logger) streamingTagValuesHandler {
+func newTagValuesStreamingGRPCHandler(cfg Config, next pipeline.AsyncRoundTripper[combiner.PipelineResponse], apiPrefix string, o overrides.Interface, logger log.Logger, dataAccessController DataAccessController) streamingTagValuesHandler {
 	postSLOHook := metadataSLOPostHook(cfg.Search.MetadataSLO)
 
 	return func(req *tempopb.SearchTagValuesRequest, srv tempopb.StreamingQuerier_SearchTagValuesServer) error {
+		ctx := srv.Context()
+		var err error
+
+		if dataAccessController != nil {
+			err = dataAccessController.HandleGRPCUnsupportedEndpoint(ctx)
```

Comment:
Should we make this a real named function and let the data access controller decide that it's unsupported and when to error?   It looks like maybe it should always be failing because it's known to be unsupported, but that isn't the case. It's only if policies are present, which is an internal concern in the data access controller.